server/src/test/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequestControllerTest.java

```diff
@@ -422,18 +410,16 @@ void testGetByCreatorTemplateRequesteeIdMultiplePermitted() {
         createDefaultRole(RoleType.PDL, pdlMemberProfileTwo);
         MemberProfile recipientOne = createADefaultRecipient();
         MemberProfile recipientTwo = createASecondDefaultRecipient();
-
-        UUID identicalTemplateId = UUID.randomUUID();
         LocalDate now = LocalDate.now();
         //create two sample feedback requests by the same PDL and same requestee
-        FeedbackRequest feedbackReq = new FeedbackRequest(UUID.randomUUID(), pdlMemberProfile.getId(), memberOne.getId(), recipientOne.getId(), identicalTemplateId, now, null, "pending", null);
+        FeedbackRequest feedbackReq = new FeedbackRequest(UUID.randomUUID(), pdlMemberProfile.getId(), memberOne.getId(), recipientOne.getId(), now, null, "pending", null);
```

Comment:
Rename this test to `testGetByCreatorRequesteeIdMultiplePermitted`?

Reply:
Fixed